docs/advanced/nat.md

```diff
@@ -1,14 +1,59 @@
 
-* **liqo.io/override-address**: the public IP address of the NAT firewall.
-* **liqo.io/override-port**: the public port of the NAT firewall.
+## Option A: Reverse Liqo network
 
-```{admonition} Tip
-In case you need to have multiple gateways behind the same NAT firewall, you need to override the port for each peer using the `--server-port` flag at peering time.
-This flag works with `liqoctl peer` and `liqoctl network connect` commands.
+![The provider is behind a NAT](../_static/images/advanced/nat/provider-nat.svg)
+
+By default using the `peer` command, the gateway server is configured on the provider cluster.
+However, there may be cases where the provider cluster's nodes are not directly reachable, such as when they are behind a NAT, while the consumer cluster is directly accessible.
+For instance, in the image above, cluster 2 is behind a NAT and is therefore not directly reachable.
+
+To fix this issue, we can swap the roles of the gateways, configuring the client on the cluster provider and the server on the consumer.
+To do so, you need to use [manual peering](./manual-peering.md), setting the inter-cluster network up separately.
+
+![The gateway server has been on the consumer side](../_static/images/advanced/nat/consumer-nat.svg)
+
+In this case, once the gateway server has been moved to cluster 1, we are able to configure the Liqo networking without the need to configure port-mapping on the NAT, as the gateway server is directly reachable.
+
+In the case you would like to keep the gateway server on the provider, you can follow the instructions on the next paragraph.
+
+## Option B: configure the gateway server to be reachable through the NAT (or physical load balancer)
+
+The solution presented in this section allows to reach the gateway server even when it is behind a NAT (or, for example, a physical load balancer) which prevents it from being directly reachable.
+
+![The provider is behind a NAT](../_static/images/advanced/nat/port-address-override.svg)
+
+In this case, **you must configure port-mapping on the NAT** so that the traffic directed on a specific port of the NAT can be forwarded to one of nodes of the cluster.
+
+**When network is configured via `liqoctl` commands**, if the service is of type `NodePort`, Liqo automatically uses the external IPs of the nodes to configure the gateway client.
+
+You can force the Liqo gateway client to use the public IP or FQDN and the port of the NAT where we configured port mapping, by providing some options to `liqoctl peer` or `liqoctl network connect` commands when using `liqoctl`.
+[With declarative peering](./peering/peering-via-cr.md#configuring-the-client-gateway-consumer-cluster) you can directly provide these values in the `GatewayClient` manifest.
+
+For example, let's imagine we expose the gateway server with a `NodePort` on port `30742` and we configure port mapping on the NAT so that port 40582 of the NAT redirects to port `30742` of a node of the cluster.
```

Comment:
```suggestion
For example, let's imagine we expose the gateway server with a `NodePort` on port `30742` and we configure port mapping on the NAT (which has public IP address equal to `203.0.113.8`) so that port 40582 of the NAT redirects to port `30742` of a node of the cluster.
```